packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -35,9 +35,12 @@ type FrameCallbackType = Deadline => void;
 type CallbackConfigType = {|
   scheduledCallback: FrameCallbackType,
   timeoutTime: number,
-  callbackId: number, // used for cancelling
+  nextCallbackConfig: CallbackConfigType | null, // creating a linked list
+  previousCallbackConfig: CallbackConfigType | null, // creating a linked list
 |};
 
+export type CallbackIdType = any;
+
```

Comment:
Why is this `any` instead of `CallbackConfigType`?

Reply:
Let's discuss this in person - the tldr is that we sometimes want the 'callbackIdType' to be a number, sometimes the `CallbackConfigType`, sometimes something else.